com.unity.template.renderstreaming-hd/Assets/Scripts/RenderStreaming.cs

```diff
@@ -211,23 +250,118 @@ void OnOffer(ISignaling signaling, DescData e)
             while (opLocalDesc.MoveNext())
             {
             }
+
             if (opLocalDesc.IsError)
             {
-                Debug.LogError($"Network Error: {opLocalDesc.Error}");
+                Debug.LogError($"Network Error: {opLocalDesc.Error.message}");
                 return;
             }
 
             signaling.SendAnswer(connectionId, desc);
         }
 
+        RTCPeerConnection PrepareNewPeerConnection(ISignaling signaling, string connectionId, bool isOffer)
```

Comment:
`PrepareNewPeerConnection` 🤔 ?
This method creates a instance of `RTCPeerConnection`, so it should be better `CreatePeerConnection` for the method name.